contracts/lbp/WrapperFactory.sol

```diff
@@ -0,0 +1,119 @@
+/*
+
+██████╗░██████╗░██╗███╗░░░███╗███████╗██████╗░░█████╗░░█████╗░
+██╔══██╗██╔══██╗██║████╗░████║██╔════╝██╔══██╗██╔══██╗██╔══██╗
+██████╔╝██████╔╝██║██╔████╔██║█████╗░░██║░░██║███████║██║░░██║
+██╔═══╝░██╔══██╗██║██║╚██╔╝██║██╔══╝░░██║░░██║██╔══██║██║░░██║
+██║░░░░░██║░░██║██║██║░╚═╝░██║███████╗██████╔╝██║░░██║╚█████╔╝
+╚═╝░░░░░╚═╝░░╚═╝╚═╝╚═╝░░░░░╚═╝╚══════╝╚═════╝░╚═╝░░╚═╝░╚════╝░
+
+*/
+
+// SPDX-License-Identifier: GPL-3.0-or-later
+
+// solium-disable linebreak-style
+pragma solidity ^0.8.4;
+
+import "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import "../utils/CloneFactory.sol";
+import "openzeppelin-solidity/contracts/access/Ownable.sol";
```

Comment:
I would import from `openzeppelin-contracts-sol8`. It's the msot recent version of OZ we use. Btw here you are using `Ownable`, so there probably is a reason why you are not using it in the `LBPWrapper` I guess.

Reply:
I'm not sure why we didn't use `Ownable`. I think there shouldn't be a good reason to add our own `onlyOwner` modifier instead of inheriting from `Ownable`, which is more extensive and secure right?

Reply:
If you can use Ownable, I would use it for the reasons you mentioned (well tested etc.)

Reply:
Because we are using Cloning. When we use Ownable the owner gets initialised in constructor. But when we clone contract, constructor is never called. Hence, owner won't be initialised and we will always face the from of 'onlyOwner'.